### PR TITLE
update readme for trt memory

### PR DESCRIPTION
**Pull request by Can-Zhao**
Fixes # .

### Description
A few sentences describing the changes proposed in this pull request.

### Status
**Ready/Work in progress/Hold**

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [ ] In-line docstrings updated.
- [ ] Update `version` and `changelog` in `metadata.json` if changing an existing bundle.
- [ ] Please ensure the naming rules in config files meet our requirements (please refer to: `CONTRIBUTING.md`).
- [ ] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [ ] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [ ] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.
- [ ] Avoid using path that contains personal information within config files (such as use `/home/your_name/` for `"bundle_root"`).

### CODE REVIEW

File: models/lung_nodule_ct_detection/docs/README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Alternatively, we provide [resampled nifti images](https://drive.google.com/driv
 ## Training configuration
 The training was performed with the following:
 
-- GPU: at least 16GB GPU memory
+- GPU: at least 16GB GPU memory, requires 24G when exporting TRT model
```
Comment (mingxin-zheng) [inline models/lung_nodule_ct_detection/docs/README.md]: We hit OOM with L40G 24GB right when exporting TRT, didn't we?

Comment (Can-Zhao) [reply models/lung_nodule_ct_detection/docs/README.md]: I changed to 32G. I thought it means it requires 24G. So it requires 32G for TRT? What is the actual requirement?

Comment (mingxin-zheng) [reply models/lung_nodule_ct_detection/docs/README.md]: cc @binliunls

Comment (binliunls) [reply models/lung_nodule_ct_detection/docs/README.md]: Hi @mingxin-zheng ,
    I ran several tests on A100/V100. The memories are different on different GPUs. The max one is a little bit greater than 24G. So I would suggest to use GPUs, whose memory is greater than (not equal to) 24G. 32G would be a nice choice.

Thanks,
Bin

**Review by mingxin-zheng — APPROVED** [review]

File: models/lung_nodule_ct_detection/docs/README.md (hunk 1 above)
Comment (mingxin-zheng) [reply models/lung_nodule_ct_detection/docs/README.md]: Thanks!

**Pull request merged**